drivers/i2c/Kconfig.stm32

```diff
@@ -34,6 +34,7 @@ config I2C_STM32_V2
 config I2C_STM32_INTERRUPT
 	bool "STM32 MCU I2C Interrupt Support"
 	depends on I2C_STM32_V1 || I2C_STM32_V2
+	default y
```

Comment:
Know it's not consistent elsewhere, but could put the `depends on` last, after the `default`. Think that might be the most common order.

Reply:
Yes, I thought about the correct order myself. Then it seemed to me this was the most common order, but maybe I was wrong.